lib/ActivityStream.jsm

```diff
@@ -212,6 +212,12 @@ const PREFS_CONFIG = new Map([
       url: "https://activity-stream-icons.services.mozilla.com/v1/messages.json.br",
       updateCycleInMs: ONE_HOUR_IN_MS * 4,
       enabled: false
+    }, {
+      id: "cfr",
+      type: "local",
+      localProvider: "CFRMessageProvider",
+      enabled: false,
+      cohort: 0
```

Comment:
Will we have different CFR cohorts?

Reply:
i _think_ so... this comment seems to suggest we will: https://bugzilla.mozilla.org/show_bug.cgi?id=1471389#c1
but it's still unclear how we're going to run this experiment with the other recommendations at the same time so for now i just left this empty, but we can do something basically like the onboarding cohort if we want